testcases/crypto/ec_func.c

```diff
@@ -420,6 +420,44 @@ CK_RV run_HMACSign(CK_SESSION_HANDLE session, CK_OBJECT_HANDLE h_key,
     return rc;
 }
 
+CK_RV run_AESEncrypt(CK_SESSION_HANDLE session, CK_OBJECT_HANDLE h_key,
```

Comment:
A general remark for tests: this test checks the correct implementation of the new derive functionality. Would it make sense to also check the correct behavior on invalid input? E.g. create a key from clear input and try to derive, etc.

Reply:
Yes, but this would be very CCA specific. I tend to keep the testcases as generic as possible....